cmd/juju/user/login.go

```diff
@@ -576,7 +576,7 @@ func (c *loginCommand) promptUserToTrustCA(ctx *cmd.Context, ctrlDetails *jujucl
 			prettyName = fmt.Sprintf("%q (%s)", host, endpoint)
 		}
 		fmt.Fprintf(ctx.Stderr, "Controller %s presented a CA cert that could not be verified.\nCA fingerprint: [%s]\n", prettyName, fingerprint)
-		trust, err := c.pollster.YN("Trust remote controller", false)
+		trust, _ := c.pollster.YN("Trust remote controller", false)
```

Comment:
Ignoring the error returned by the pollster instance is the correct behavior here. If the user does not type `Y`, pollster returns `false` (and an error which doesn't really matter) causing the if block below to abort the connection attempt with an error.

Reply:
Maybe a code comment would help the next reader of the code